imports/ui/modules/common/modal/GeneralModal.jsx

```diff
@@ -44,6 +44,10 @@ class GeneralModal extends React.PureComponent {
     const title = _get(options, 'title') || MODAL_WIZARDS_DEFAULT_TITLE[modalWizard];
     const icon = _get(options, 'icon') || MODAL_WIZARDS_DEFAULT_ICON[modalWizard];
     const ModalComponent = ListWizard[modalWizard];
+    let cssClass = '';
+    if (modalWizard === MODAL_WIZARDS.ADD_CUSTOMER || modalWizard === MODAL_WIZARDS.EDIT_CUSTOMER) {
+      cssClass = 'customer-modal';
+    }
```

Comment:
Remove the cssClass

Reply:
```suggestion
   let width= 520;
    if (modalWizard === MODAL_WIZARDS.ADD_CUSTOMER || modalWizard === MODAL_WIZARDS.EDIT_CUSTOMER) {
      width= 650;
    }
```
Change `cssClass ` into  `width` property